modules/tracker/database/mysql/2.0.0.sql

```diff
@@ -0,0 +1,118 @@
+-- Midas Server. Copyright Kitware SAS. Licensed under the Apache License 2.0.
+
+-- MySQL database for the tracker module, version 1.2.4
+
+CREATE TABLE IF NOT EXISTS `tracker_producer` (
+    `producer_id` bigint(20) NOT NULL AUTO_INCREMENT,
+    `community_id` bigint(20) NOT NULL,
+    `repository` varchar(255) NOT NULL,
+    `executable_name` varchar(255) NOT NULL,
+    `display_name` varchar(255) NOT NULL,
+    `description` text NOT NULL,
+    `revision_url` text NOT NULL,
+    PRIMARY KEY (`producer_id`),
+    KEY (`community_id`)
+) DEFAULT CHARSET=utf8;
+
+CREATE TABLE IF NOT EXISTS `tracker_scalar` (
+    `scalar_id` bigint(20) NOT NULL AUTO_INCREMENT,
+    `trend_id` bigint(20) NOT NULL,
+    `value` double,
+    `submission_id` bigint(20) NOT NULL DEFAULT '-1',
+    PRIMARY KEY (`scalar_id`),
+    KEY (`trend_id`),
+    KEY (`submission_id`)
+) DEFAULT CHARSET=utf8;
+
+CREATE TABLE IF NOT EXISTS `tracker_submission` (
+    `submission_id` bigint(20) NOT NULL AUTO_INCREMENT,
+    `producer_id` bigint(20) NOT NULL,
+    `name` varchar(255) NOT NULL DEFAULT '',
+    `uuid` varchar(255) NOT NULL DEFAULT '',
```

Comment:
Out of curiosity I checked if you can have `uuid()` as a default value in mysql.  You can't, but you could set a uuid through a trigger on inserts.  I'm only reporting this, not suggesting a change here.